emapper.py

```diff
@@ -21,9 +22,11 @@
 from eggnogmapper import search
 from eggnogmapper import annota
 from eggnogmapper import seqio
+from eggnogmapper import orthology
 from eggnogmapper.utils import colorify
 from eggnogmapper.server import (server_functional, load_server,
                                  generate_idmap, shutdown_server)
+from ete3 import NCBITaxa
```

Comment:
this was after the line `if args.guessdb:`. Keep it there, otherwise it becomes a mandatory dependency for the program